installd/MIDaemonConfiguration.h

```diff
@@ -0,0 +1,14 @@
+#import "MIGlobalConfiguration.h"
+
+API_AVAILABLE(ios(13.0))
+@interface MIDaemonConfiguration : MIGlobalConfiguration
+
+@property (nonatomic,readonly) BOOL skipDeviceFamilyCheck; 
+@property (nonatomic,readonly) BOOL skipThinningCheck; 
+@property (nonatomic,readonly) BOOL allowPatchWithoutSinf; 
+
+- (BOOL)skipDeviceFamilyCheck;
+- (BOOL)skipThinningCheck;
+- (BOOL)allowPatchWithoutSinf;
```

Comment:
You don't need these lines because you have already declared the properties at line 6 - 8